internal/service/streamprocessor/data_source.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
 )
 
-const StreamProccesorName = "streamprocessor"
+const StreamProccesorName = "stream_processor"
```

Comment:
```suggestion
const StreamProcessorName = "stream_processor"
```
typo in the const name, should also change where it is used

Reply:
fixed in f171b0c8da92befc18cd794a37bd8e85fd90a796